src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileDownloadMenuItem.java

```diff
@@ -112,9 +112,9 @@ public void configure(EntityBundle bundle, RestrictionInformationResponse restri
 					if (directDownloadUrl.startsWith(sftpProxy)) {
 						view.setIsAuthorizedDirectDownloadLink();
 						loginModalWidget.configure(directDownloadUrl, FormPanel.METHOD_POST, FormPanel.ENCODING_MULTIPART);
```

Comment:
You could delete LoginModalWidget (it was only used for this sftp authenticated download).  Recommend you change this to simply popupUtilsView.showInfoDialog(...)